src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -320,7 +320,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }, [
         workspace && h(WrappedComponent, {
           ref: child,
-          workspace, refreshWorkspace, refreshRuntimes, runtimes, persistentDisks, galaxyDataDisks,
+          workspace, refreshWorkspace, refreshRuntimes, refreshApps, runtimes, persistentDisks, galaxyDataDisks,
```

Comment:
Unrelated to PPW, but I think this change fixes https://broadworkbench.atlassian.net/browse/IA-2917. Previously `refreshApps` was not being passed to the `Notebooks` child component, causing the `Cannot read property `apply` of undefined` error.